### DISCUSSION TITLE
FR: Community-Based Abandonware Report System

### DISCUSSION THREAD

**Discussion by OracleHisty** [modrinth/.github#282] [Feature requests]
A social issue with Minecraft modding is that at some point a developer may just totally burn out, and neglect to tell their community.

 On the off chance that this is the case, having a community - centered filter system to report a potentially abandoned mod or project, where at a certain number of Abandonware reports will flag the project as "User Reports Suggest Potentially Abandoned" 

If this were just a visual notice, then no extra features would be needed, however it might be plausible to create a system to notify the developer through their attached email after a certain number of reports saying: 

"Project is being reported as abandoned by Modrinth userbase, do you want to flag your project as abandoned/archived?"

And this could also extend to mods with a publicly available source to potentially get new maintainers for Abandonware mods. 

A prime example of an already solved mod in a similar way:
Dimensional Doors has gone from closed as original due to old modding standards, to open and Accepting Pull Requests from the community occasionally. There even is a Legacy Maintainers for the older version of Minecraft! In dimdoors case, the original creator had life to deal with, as any person would at some point. 

I feel for Abandonware mods, the community around them would be more than happy to contribute at some point towards maintaining their beloved mods of which have been abandoned due to some extraneous circumstances.

An example of a mod that would benefit from this system is Grimoire of Gaia. The creator has long since burnt out over time, resulting in an unmaintained project, despite a formerly bustling community around it.

On the rare chance that a mod was abandoned due to an unfortunate passing of the creator/maintainer (as much as nobody really wants this to happen, it's part of life) this would be an excellent system to potentially mark a project for potentially open to fresh hands or remakes for mods that lose their userbase due to it.

**Comment by Minenash** [comment]
Assuming the license allows it (and if not there's ways to context authors outside of modrinth), if a project is abounded, then someone else can update it / add features to it and upload it as another project. Or if the author agrees, they could always add people to their project.

And I don't think a banner would be helpful
